modules/microvm/virtualization/microvm/audiovm.nix

```diff
@@ -57,6 +57,7 @@
             pkgs.pulseaudio
             pkgs.pamixer
             pkgs.pipewire
+            pkgs.alsa-utils
```

Comment:
Are the `alsa-utils` required?

Reply:
It's very convenient to use aplay and speaker-test from alsa-utils to test USB headsets until we have the audio control application merged. I've updated it to enable this conditionally when debug is on.